features/registration.feature

```diff
@@ -101,4 +101,4 @@ Feature: Booting VM with various registration settings
     Examples:
       | provider   |
       | virtualbox |
-      | libvirt    |
+  #    | libvirt    |
```

Comment:
I commented it out temporarily and forgot about it. I enable it again since it can as well be controlled via the env variables. Good catch.